core/tee/tee_rpmb_fs.c

```diff
@@ -1112,7 +1112,7 @@ static TEE_Result tee_rpmb_init(uint16_t dev_id)
 
 		res = tee_rpmb_verify_key_sync_counter(dev_id);
 		if (res == TEE_ERROR_ITEM_NOT_FOUND &&
-			!rpmb_ctx->key_verified) {
+			!rpmb_ctx->key_verified && plat_rpmb_ready()) {
```

Comment:
I think this function call should be moved to `tee_rpmb_write_and_verify_key()`.